src/main/java/frc/robot/subsystems/swerve/SwerveSS.java

```diff
@@ -62,7 +62,7 @@ public void drive(Translation2d translation, double rotation, boolean fieldRelat
                 )
         );
         // making sure the speeds are within the max speed
-        SwerveDriveKinematics.normalizeWheelSpeeds(swerveModuleStates, SwerveConstants.MAX_SPEED);
+        SwerveDriveKinematics.desaturateWheelSpeeds(swerveModuleStates, SwerveConstants.MAX_SPEED);
```

Comment:
you sure this is the same function?

Reply:
![image](https://user-images.githubusercontent.com/29277997/152158356-6030a8a1-b9dd-4110-9696-0e6d9b994f81.png)